### ISSUE TITLE
Add Uninstall guide for Mroonga

### ISSUE THREAD

**Issue by ryfjwr** [mroonga/mroonga.org#12]
# Background

- There is only [install guide](http://mroonga.org/ja/docs/install.html) for Mroonga at Official site.
-  But, no document exists when developer uninstalls Mroonga, without removing MySQL or  MariaDB.

# Environment

- OS: Linux(as for me, CentOS 6.5)
- Installed MariaDB or MySQL(for me, MariaDB `10.0.13`)
- Installed Mroonga by source code.
  - for me, [this way](http://mroonga.org/ja/docs/install/others.html#build-from-source).
- Used Mroonga by MySQL's Plugin. 

# Expectation

- I propose publishing uninstall Mroonga guide for developer who already installed Mroonga.

# Origin 

- [this workshop](https://github.com/oss-gate/workshop/issues/333#issuecomment-289191212)

**Comment by kou** [comment]
Thanks for your report!

Here is the way how to uninstall Mroonga installed from source:

```text
% mysql < ${PREFIX}/share/mroonga/uninstall.sql
% cd ${MROONGA_BUILD_DIR}
% sudo make uninstall
```

Could you try to add the way to the documentation: https://github.com/mroonga/mroonga/blob/master/doc/source/install/others.rst

http://mroonga.org/ja/docs/contribution/documentation.html will help you.

**Comment by komainu8** [comment]
I close this issue.
Because this issue has already fixed by #135.
